jenkins/internal-ci/generic/branch/release.groovy

```diff
@@ -125,15 +125,31 @@ pipeline {
         }
         
         stage ('Sign rpm') {
-            when { expression { false } }
+            when { expression { true } }
             steps {
                 script { build_stage = env.STAGE_NAME }
                                 
                 sh label: 'Generate Key', script: '''
                     set +x
                     pushd scripts/rpm-signing
-                    cat gpgoptions >>  ~/.rpmmacros
-                    sed -i 's/passphrase/'${passphrase}'/g' genkey-batch
+                    for expiredKey in $(gpg2 --list-keys | grep -B1 "Seagate"|head -1|tr -d " ");
```

Comment:
Is there a chance to have multiple Seagate signed GPG Keys on the container?

Reply:
No chance to have multiple Seagate key with same name

Reply:
In that case, can we remove for loop and just use a single command?

Reply:
Yes I have removed for loop from code.